Makefile

```diff
@@ -0,0 +1,6 @@
+all: image
+	docker-compose up --scale raft=3 -V
+bin: 
+	cargo build --release
+image: bin
+	docker build -t marinpostma/meilisearch .
```

Comment:
you can use `docker build -t marinpostma/meilisearch -f Dockerfile.raft .` and avoid Dockerfile renaming tricks

Reply:
thanks for the tip!!